vosekast_control/Tank.py

```diff
@@ -187,3 +187,8 @@ def state(self):
     def state(self, new_state):
         self._state = new_state
         self.logger.info(f"New Tank state is: {new_state}")
+
+        mqttmsg = StatusMessage(
+                    self.name, f"New Tank state is: {new_state}", None, None, None)
```

Comment:
Has a status message not a clear value? That looks for me like a log message.

Reply:
changed to send log message only